<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="login_label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>1</y>
      <width>81</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="email_login">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>102</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Email / UserName</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_login">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>163</y>
      <width>361</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sign_in_login_btn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>256</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Sign In</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="show_pass_login">
    <property name="geometry">
     <rect>
      <x>33</x>
      <y>219</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="forgot_login">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>319</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Forgot</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pass_forgot_login">
    <property name="geometry">
     <rect>
      <x>183</x>
      <y>319</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="question">
    <property name="geometry">
     <rect>
      <x>271</x>
      <y>326</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>?</string>
    </property>
   </widget>
   <widget class="QLabel" name="dont_acc_login">
    <property name="geometry">
     <rect>
      <x>92</x>
      <y>357</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Dont Have An Account?</string>
    </property>
   </widget>
   <widget class="QPushButton" name="signup_btn_login">
    <property name="geometry">
     <rect>
      <x>263</x>
      <y>355</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Sign Up</string>
    </property>
   </widget>
   <widget class="QLabel" name="time_rem_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>55</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>422</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
